trade-finance-manager-api/src/v1/helpers/amendment.helpers.js

```diff
@@ -205,6 +209,39 @@ const sendManualBankDecisionEmail = async (amendmentVariables) => {
   }
 };
 
+/**
+ * Evaluated whether facility amendment is eligible
+ * for ACBS interaction based on myriads of conditions.
+ * This function evaluated across all amendment types.
+ * @param {Object} amendment Facility amendments object
+ */
+const canSendToAcbs = (amendment) => {
+  // Ensure at least one of the attribute has been amended
+  const hasBeenAmended = amendment.changeCoverEndDate || amendment.changeFacilityValue;
+  // Amendment status is marked as `Completed`
+  const completed = amendment.status === AMENDMENT_STATUS.COMPLETED;
+  // Amendment has been submitted by PIM team
+  const PIM = Boolean(amendment.submittedByPim);
+  // Manual amendment verification
+  const manual = Boolean(amendment.requireUkefApproval) && Boolean(amendment.bankDecision);
+
+  // Manual amendment
+  if (manual) {
+    // Bank Decision
+    const { submitted, decision } = amendment.bankDecision;
+    // Bank has accepted the UW decision
+    const proceed = decision === AMENDMENT_BANK_DECISION.PROCEED;
+    // Ensure not all of the amendment requests are declined
+    const { value, coverEndDate } = amendment.ukefDecision;
+    const declined = value === AMENDMENT_UW_DECISION.DECLINED && coverEndDate === AMENDMENT_UW_DECISION.DECLINED;
```

Comment:
will this work if the amendment only has one change? and will it not crash trying to match undefined to declined?

Reply:
Yes this works, have tested a manual amendment.
`undefined` comparison should not cause the script to crash it will simply be `false`.